department-of-veterans-affairs/content-build
# VACMS-18464 Show all topics by default on Resources & Support landing page

## Summary
The new Family Member and Caregiver Benefit Hub was recently published and we want users to be able to see the links on the Resources & Support landing page (under "Browse by topic").

Currently, that section is truncated down to 6 link lists, so we are removing the truncation and the "Show all topics" button completely from the Lists of Links component.

It was determined that the Resources & Support landing page is the only prod page where this component is used so we can safely modify it for that route specifically (`/resources`). 

Ticket comments for context:
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18464#issuecomment-2272132968
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18464#issuecomment-2276409136

## Related issue(s)
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18464

## Testing done
**This can be tested on Tugboat**: https://web-w4azz2khpzdi53uymp1gxkpi0fbqdlqu.demo.cms.va.gov/resources/

1. Check `/resources` locally to be sure there are link lists matching the selected categories in Drupal for this page:

<img width="600" alt="Screenshot 2024-08-05 at 1 34 31 PM" src="https://github.com/user-attachments/assets/b888dc56-12c6-476a-92ac-fd6745a1d3de">

2. Be sure sort order (alphabetically) of categories is preserved; note that links within each category are not alphabetized and should match the order shown in prod
3. Click some (or all) of the links within the sections to be sure link behavior is preserved
4. Validate that there is no "Show all topics" button on `/resources`
5. Check mobile for the above steps

## Screenshots

<details><summary>DESKTOP - Resources & Support landing page</summary>
<img width="812" alt="Screenshot 2024-08-05 at 2 10 38 PM" src="https://github.com/user-attachments/assets/840b8b92-7feb-4c0a-b690-5f1c36428bc3">
</details>


<details><summary>MOBILE - Resources & Support landing page</summary>
<img width="400" alt="Screenshot 2024-08-05 at 1 47 54 PM" src="https://github.com/user-attachments/assets/64b7f678-ae5a-495c-b3e4-d3426922093d">
<img width="400" alt="Screenshot 2024-08-05 at 1 48 07 PM" src="https://github.com/user-attachments/assets/cf360e21-f30e-4812-83d1-7b6a92115768">
<img width="400" alt="Screenshot 2024-08-05 at 2 11 00 PM" src="https://github.com/user-attachments/assets/d89f7a27-bb1b-4551-9ff6-240cd9388bc8">
</details>

## What areas of the site does it impact?

Resources & Support landing page (`/resources`)

## Acceptance criteria

- [x] New topic has capability of being shown without hidden content shown after button is clicked
- [x] Sort order is preserved
